CHANGES.rst

```diff
@@ -16,6 +16,7 @@ Breaking Changes
   ``False``.
 - ``RequestOptions.auto_parse_qs_csv`` now defaults to ``False`` instead of
   ``True``.
+  ``API.independent_middleware`` now defaults to ``True`` instead of ``False``.
```

Comment:
IIRC this is actually a kwarg on ``API.__init__()``, as opposed to a property. Also, would you mind adding a note to ``docs/changes/2.0.0.rst`` as well?